src/cli/api_client.rs

```diff
@@ -296,14 +296,25 @@ impl ApiClient {
         tx_priority: TxPriority,
         consolidate_deprecated_keychains: bool,
         interval_trigger: Option<u32>,
+        manual_trigger: Option<bool>,
     ) -> anyhow::Result<()> {
         let tx_priority = match tx_priority {
             TxPriority::NextBlock => proto::TxPriority::NextBlock as i32,
             TxPriority::HalfHour => proto::TxPriority::HalfHour as i32,
             TxPriority::OneHour => proto::TxPriority::OneHour as i32,
         };
-
-        let trigger = interval_trigger.map(proto::payout_queue_config::Trigger::IntervalSecs);
+        let trigger = match (interval_trigger, manual_trigger) {
+            (Some(interval), None) | (Some(interval), Some(false)) => {
+                Some(proto::payout_queue_config::Trigger::IntervalSecs(interval))
+            }
+            (None, Some(true)) => Some(proto::payout_queue_config::Trigger::Manual(true)),
+            (Some(_), Some(true)) => {
+                return Err(anyhow::anyhow!(
```

Comment:
This can be enforced via clap https://docs.rs/clap/latest/clap/struct.ArgGroup.html. I don't know exactly how it translates to the derived syntax but it's possible https://docs.rs/clap/latest/clap/_derive/index.html#arggroup-attributes.

Can be a follow up to figure that out. In that case we can change this branch to `unreachable!()` as clap should prohibit ever getting here.

Reply:
I tried `conflicts_with` earlier but that did not work as expected for the use case so wrote a manual implementation